docs/overview/04-proof-aggregation/04-domain-management.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Domain Management
+---
+
+A domain is identified by an identifier and identifies a class of aggregation and its key properties
+
+- Aggregation size
+- Publishing queue size
+
+Everyone can create a new domain and become the owner. The owner should hold some founds in order to create a new domain, the hold amount depends on the size of both aggregation and queue that the domain define and these found will be released when the domain is removed.
+
+## Create a new domain
+
+To create a new domain the owner should call [`registerDomain(aggregationSize, queueSize)`](../02-mainchain/05-mainchain_api.md#registerdomain) extrisic and define both aggregation size and queue size.
+
+The aggregation size should be any unsigned integer smaller than 128, but because it represents a Merkle tree size is better to use a power of two. From the aggregation size a proof submitter that would use this domain can estimate what will be the cost of check his proof presence in the aggregation receipt in the destination chain where he wants to use his proof: a bigger size means a bigger cost on check it, but also a lower cost send the proof on the destination chain due the bridge cost is amortized by more proofs in the same attestation.
+
+The publishing queue size should be an unsigned integer smaller or equal than 16 and represent how many aggregations can wait for publication: a lower number can generate a more common `CannotAggregate(DomainFull)` event on submitting proof because there is not enough buffer in the publication queue, on the other side use a bigger aggregation size means hold more money.
+
+If the owner has not enough founds to cover the storage cost needed to maintain the domain the `registerDomain` call fails, otherwise the founds are put on hold for all domain life.
```

Comment:
```suggestion
The publishing queue size should be an unsigned integer smaller or equal than 16 and represents how many aggregations can wait for publication: a smaller number can generate a more common `CannotAggregate(DomainFull)` event on proof submission because there is not enough buffer in the publication queue, on the other side using a bigger aggregation size means hold more money.

If the owner has not enough funds to cover for the storage cost needed to maintain the domain the `registerDomain` call fails, otherwise the funds are put on hold for all the domain's life.
```